core/src/main/scala/com/salesforce/op/evaluators/OpEvaluatorBase.scala

```diff
@@ -61,28 +61,28 @@ trait OpHasLabelCol[T <: FeatureType] extends Params {
 /**
  * Trait for predictionCol which contains all output results param
  */
-trait OpHasFullPredictionCol extends Params {
-  final val fullPredictionCol: Param[String] = new Param[String](this, "fullPredictionCol", "prediction column name")
+trait OpHasPredictionCol extends Params {
+  final val predictionCol: Param[String] = new Param[String](this, "fullPredictionCol", "prediction column name")
```

Comment:
Update name and doc of the param?
Add `setDefault`?

Reply:
there is no default that can be set for this